site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -2,4 +2,31 @@
 api: offscreen
 ---
 
-<!-- Intentionally blank -->
+## Manifest
+You must declare the "offscreen" permission in the [extension manifest][1] to use the input.ime API. For example:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "offscreen"
+  ],
+  ...
+}
+```
+Offscreen documents are treated as web pages, and do not share the same permissions as service workers. Offscreen Windows cannot be focused, and cannot have their opener set using the [chrome.windows API](docs/extensions/reference/windows/) method `windows.setSelfAsOpener`.
```

Comment:
> Offscreen documents are treated as web pages

This isn't quite right. Offscreen documents are notably different from both typical web page and extension pages. Extensions can't load arbitrary page as an offscreen document - they have to be bundled in the extension's package. But they're also different from typical pages packed in an extension because they have very limited access to extensions APIs. 

> and do not share the same permissions as service workers

They have the same permissions. For example, the "clipboardWrite" permission applies to offscreen documents, meaning extensions can call `document.execCommand('copy')` in an offscreen document.

This suggestion is very much that: a suggestion. Feel free to use it in part, whole, or completely rewrite it.

```suggestion
Pages loaded as offscreen documents are handled differently from other types of extensions pages. The extension's permissions carry over to offscreen documents, but extension API access is heavily limited. Currently, an offscreen document can only use the `chrome.runtime` APIs to send and receive messages; all other extension APIs are not exposed. Other notable differences between offscreen documents and normal pages are as follows:

* Offscreen document's URL must be a static HTML file bundled with the extension.
* Offscreen documents cannot be focused.
* Offscreen documents cannot have their `opener` property set using the [`chrome.windows` API](docs/extensions/reference/windows/) method `windows.setSelfAsOpener`.
```